recipes-extended/xen/files/libxl-const-stubdom-is-linux.patch

```diff
@@ -0,0 +1,11 @@
+--- a/tools/libxl/libxl_internal.h
```

Comment:
This patch needs some kind of header, obviously the description is trivial but specifying the removal plan is important.  I would imagine this is very unstreamable.

Reply:
I already upstreamed it:
commit e7cfcdc6719d586eb7cdb62d40275a7d17fe6760
Author: Jason Andryuk <jandryuk@gmail.com>
Date:   Wed Mar 30 14:17:22 2022 -0400

    libxl: constify libxl__stubdomain_is_linux

Reply:
Sorry, it is needed - it will only be in 4.17.  I updated to use the full `git format-patch` output as the patch and mention 4.17 in the commit message.

Reply:
:+1: will expect to drop it when we go to 4.17.